test/integration-test-wrapper.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/python3
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+'''Test wrapper command for driving integration tests.
+
+Note: This is deliberately rough and only intended to drive existing tests
+with the expectation that as part of formally defining the API it will be tidy.
+
+'''
+
+import argparse
+from pathlib import Path
+import shlex
+import subprocess
+import sys
+import textwrap
+
+
+EMERGENCY_EXIT_DROPIN = """\
+[Unit]
+Wants=emergency-exit.service
+"""
+
+
+EMERGENCY_EXIT_SERVICE = """\
+[Unit]
+DefaultDependencies=no
+Conflicts=shutdown.target
+Conflicts=rescue.service
+Before=shutdown.target
+Before=rescue.service
+FailureAction=exit
+
+[Service]
+ExecStart=false
+"""
+
+
+def main():
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument('--meson-source-dir', required=True)
+    parser.add_argument('--meson-build-dir', required=True)
+    parser.add_argument('--test-name', required=True)
+    parser.add_argument('--test-number', required=True)
+    parser.add_argument('mkosi_args', nargs="*")
+    args = parser.parse_args()
+
+    test_unit = f"testsuite-{args.test_number}.service"
+
+    dropin = textwrap.dedent(
+        """\
+        [Unit]
+        After=multi-user.target network.target
+        Requires=multi-user.target
+
+        [Service]
+        Type=idle
+        StandardOutput=journal+console
+        """
+    )
+
+    if not sys.stdin.isatty():
+        dropin += textwrap.dedent(
+            """
+            [Unit]
+            SuccessAction=exit
+            FailureAction=exit
+            """
+        )
+
+        journal_file = (Path(args.meson_build_dir) / (f"test//journal/{args.test_name}.journal")).absolute()
```

Comment:
Is the `//` intentional?

```suggestion
        journal_file = (Path(args.meson_build_dir) / (f"test/journal/{args.test_name}.journal")).absolute()
```